### DISCUSSION TITLE
Rebrand - Part 1

### DISCUSSION THREAD

**Discussion by Pukimaa** [discordexperimenthub/deh-assets#9] [Designs]
Hey everyone,

My name is Pukima and I am the original designer of most branding parts you see at the Discord Experiment Hub. I am planning big things together with @Tolga1452 to make the server even better. ✨

Our first step is a rebrand. I know what you might say..."But the branding was fine" and all that. But for now we don't plan on changing a lot. However I am currently experimenting with new colors, but that's another discussion and we won't change that until we and you are truly happy with it.

Also took quite some time redoing all the organization structure in my designing program. Everything is extremely modular and organized, so it becomes a breeze changing things around. Will also probably release the source file at one point, in order to make it even easier for you to make your own DEH Assets right within Figma. This repo will have the new changes when I get to it...also wanna redo the structure here, to make it easier to find the right assets.

We are really looking forward to these changes and would love to hear your feedback! 🎉

#### New Logo
The simple logo is great. But it still bugged me a bit so I made some adjustments to make it feel more modern and also giving it more personality that will be used throughout the new branding.
##### Old Logo
<img src="https://github.com/discordexperimenthub/deh-assets/assets/58347116/15ad467c-4280-4c2e-ac7b-8b2e4341de5c" width="256px" />

##### New Logo
<img width="256" src="https://github.com/discordexperimenthub/deh-assets/assets/58347116/98468c2a-751f-4d5d-a910-fc9918721740"><br />
#### New Font Usage
Due to copyright issues and making some users think DEH is an official server by Discord we decided to change the font. After careful selection we picked out "Switzer".
#### New Banner
Due to the changes to banners with Tabs V2 and the branding changes highlighted earlier we decided to alter it a bit.
![Banner](https://github.com/discordexperimenthub/deh-assets/assets/58347116/c8397dac-3e39-4d06-a888-8b1385e9f325)
#### New Invite Splash
We might keep the original one for now (you can decide), since I just wanted a recreation with slightly updated icons in order to make it easy to change around colors and have it in higher resolution, if needed.
![Invite Splash](https://github.com/discordexperimenthub/deh-assets/assets/58347116/b5a16bbc-907e-45f7-8e5f-59281a6fa37e)

**Comment by ghost** [comment]
lgtm

**Reply by Pukimaa** [reply]
Glad you like it! 😄

**Comment by Pukimaa** [comment]
Changed the font to "Galano Grotesque"

**Reply by Tolga1452** [reply]
do not forget to add them to the repo ^^ also i think i forgot the new dev logo

**Reply by Pukimaa** [reply]
Yeah. Wanna restructure the whole repo :)
